docs/Installation.md

```diff
@@ -68,45 +68,48 @@ sudo systemctl reload httpd
 
 ### 2. Debian
 
-#### Install apache2 and disable default site
+#### Install Apache
 
 ```sh
 sudo apt-get update && sudo apt-get upgrade -y 
-sudo apt-get install apache2 unzip
+sudo apt-get install -y apache2 unzip
+```
+
+#### Basic Apache configuration
+
+```sh
 sudo a2enmod proxy
 sudo a2enmod proxy_http
 sudo a2enmod rewrite
```

Comment:
This can be done in one command :
```
sudo a2enmod proxy proxy_http rewrite
```

Reply:
I thought that didn't work, but it seems it does!